src/main/java/com/shrralis/ssdemo1/service/interfaces/IUserService.java

```diff
@@ -15,4 +15,16 @@
 	List<User> getAllUsers();
 
 	User getUser(int id);
+
+    User findById(Integer id);
+
+    User findByLogin(String login);
+
+    List<User> findByLoginOrEmailContainingAllIgnoreCase(String login, String email);
```

Comment:
Can you convert this madness to `@Query`?

Reply:
why would i? it perfectly conveys the behavior of the method. have you seen spring api? they love incredibelly long classes'&packages' names because they are extremely descriptive. besides, java a priori is a pretty verbose programming language

Reply:
It's too scary... But nevermind